base/operators.jl

```diff
@@ -3,10 +3,27 @@
 ## types ##
 
 """
-    <:(T1, T2)
+```julia
+<:(L, R)
+```
+
+The subtyping predicate: test whether the type `L` may be a subtype of `R`.
+
+For any type `A` and any type `B`, `A` subtypes `B` if and only if any value
+belonging to `A` is also of type `B`. Subtyping is reflexive and transitive,
+making it a *preorder*.
+
+Julia is sometimes not able to prove a subtyping relation between two types, even
+if such a relation holds. In such a case the result of `S <: T` will be `false`
+even if `S` subtypes `T`. Even so, `<:` is expected to be reflexive and transitive.
+
+See also: [the Manual page on types](@ref man-types), [`Union{}`](@ref),
+[`Any`](@ref), [`isa`](@ref).
 
-Subtype operator: returns `true` if and only if all values of type `T1` are
-also of type `T2`.
+Julia's subtyping relation is the topic of the paper *Julia Subtyping: A Rational
```

Comment:
Ordinary style is for citations to go at the end of the docstring.

Reply:
As much as I like that paper, I am not sure it is a good idea to cite it in the Julia docs. It addressed a previous state of the type system and many suggestions in the paper have been acted upon.

Reply:
True, it's not up-to-date anymore for current Julia. I think you may be right, but there's some things in that paper that should maybe appear in the Julia docs, like Table 1, *Julia subtyping compared*.